sick_scan2: # configuration for sick_scan2_lms_5xx: For compatibility with ros versions previous to foxy, node and parameter name changed to sick_scan2 for all supported scanner. The type of scanner is configured by scanner_name only.
  ros__parameters:
    hostname : "192.168.0.1"
    frame_id : "cloud"
    scanner_name : "sick_lms_5xx"
    port : 2112
    min_ang : -1.658062789 #in lidar cordinate frame  see ../doc/3d_coordinate_system_comp.png
    max_ang : 1.658062789 #in lidar cordinate frame  see ../doc/3d_coordinate_system_comp.png
    max_range : 80.0 #in meters
